docs/api/ee/query-reference-validation.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Query reference validation"
+summary: |
+  API endpoints for Query reference validation.
+---
+
+# Query reference validation
+
+API endpoints for Query reference validation.
+
+## `GET /api/ee/query-reference-validation/invalid-cards`
+
+List of cards that have an invalid reference in their query. Shape of each card is standard, with the addition of an
+  `errors` key. Supports pagination (`offset` and `limit`), so it returns something in the shape:
+
+  ```
+    {:total  200
+     :data   [card1, card2, ...]
+     :limit  50
+     :offset 100
+  ```.
```

Comment:
The period after the code snippet is messing up markdown

Reply:
This should be handled now.